ui/tests/unit/models/pki/config-test.js

```diff
@@ -0,0 +1,13 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'vault/tests/helpers';
+
+module('Unit | Model | pki/config', function (hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
```

Comment:
Can this comment be removed or is it a future todo to make this unit test more comprehensive?